torch/_inductor/fx_passes/b2b_gemm.py

```diff
@@ -99,9 +105,71 @@ def b2b_gemm_grid(M, P, meta):
 )
 
 
-B2B_GEMM_PASS = PatternMatcherPass(
-    pass_name="b2b_gemm_pass",
-)
+# numbers of loads and stores in baseline and b2bgemm
```

Comment:
style nit: Can we use the `"""` style comments, e.g.,

```
def foo(self):
    """
    This is a function that does something.
    """
```

Reply:
Updated.